Doc/whatsnew/3.14.rst

```diff
@@ -80,495 +80,435 @@ plus syntax highlighting in the REPL,
-* :ref:`PEP 750: Template strings <whatsnew314-pep750>`
-* :ref:`PEP 758: Allow except and except* expressions without parentheses <whatsnew314-pep758>`
-* :ref:`PEP 761: Discontinuation of PGP signatures <whatsnew314-no-more-pgp>`
-* :ref:`PEP 765: Disallow return/break/continue that exit a finally block <whatsnew314-finally-syntaxwarning>`
+Interpreter improvements:
+
+* :pep:`649` and :pep:`749`: :ref:`Deferred evaluation of annotations
+  <whatsnew314-deferred-annotations>`
+* :pep:`734`: :ref:`Multiple interpreters in the standard library
+  <whatsnew314-multiple-interpreters>`
+* :pep:`750`: :ref:`Template strings <whatsnew314-template-string-literals>`
+* :pep:`758`: :ref:`Allow except and except* expressions without brackets
+  <whatsnew314-bracketless-except>`
+* :pep:`765`: :ref:`Control flow in finally blocks
+  <whatsnew314-finally-syntaxwarning>`
+* :pep:`768`: :ref:`Safe external debugger interface for CPython
+  <whatsnew314-remote-debugging>`
+* :ref:`A new type of interpreter <whatsnew314-tail-call-interpreter>`
 * :ref:`Free-threaded mode improvements <whatsnew314-free-threaded-cpython>`
-* :ref:`PEP 768: Safe external debugger interface for CPython <whatsnew314-pep768>`
-* :ref:`PEP 784: Adding Zstandard to the standard library <whatsnew314-pep784>`
-* :ref:`A new type of interpreter <whatsnew314-tail-call>`
-* :ref:`Syntax highlighting in the default interactive shell <whatsnew314-pyrepl-highlighting>`,
-  and color output in :ref:`unittest <whatsnew314-color-unittest>`,
-  :ref:`argparse <whatsnew314-color-argparse>`,
-  :ref:`json <whatsnew314-color-json>` and
-  :ref:`calendar <whatsnew314-color-calendar>` CLIs
-* :ref:`Binary releases for the experimental just-in-time compiler <whatsnew314-jit-compiler>`
+* :ref:`Improved error messages <whatsnew314-improved-error-messages>`
+
+Significant improvements in the standard library:
+
+* :pep:`784`: :ref:`Zstandard support in the standard library
+  <whatsnew314-zstandard>`
+* :ref:`whatsnew314-asyncio-introspection`
+* :ref:`whatsnew314-concurrent-warnings-control`
+* :ref:`Syntax highlighting in the default interactive shell
+  <whatsnew314-pyrepl-highlighting>`, and color output in several
+  standard library CLIs
+
+C API improvements:
+
+* :pep:`741`: :ref:`Python configuration C API <whatsnew314-capi-config>`
+
+Platform support:
+
+* :pep:`776`: Emscripten is now an :ref:`officially supported platform
+  <whatsnew314-build-changes>`, at :pep:`tier 3 <11#tier-3>`.
+
+Release changes:
 
+* :pep:`779`: :ref:`Free-threaded Python is officially supported
+  <whatsnew314-free-threaded-now-supported>`
+* :pep:`761`: :ref:`Discontinuation of PGP signatures <whatsnew314-no-more-pgp>`
+* :ref:`Windows and macOS binary releases now support the experimental
+  just-in-time compiler <whatsnew314-jit-compiler>`
+* :ref:`Binary releases for Android are now provided <whatsnew314-build-changes>`
 
 
-New features
+New Features
 ============
```

Comment:
Use sentence case for headers:

https://devguide.python.org/documentation/style-guide/#capitalization

```suggestion
New features
============
```